recipes/fenicsprecice/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "fenicsprecice" %}
+{% set version = "1.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fenicsprecice-{{ version }}.tar.gz
```

Comment:
Why do you need this here instead of the [official release on GitHub](https://github.com/precice/fenics-adapter/releases)? Should we change something in the releases to be more fitting for packaging?

Reply:
Using the pip package is the default for conda if a pip package is available, as it simplifies the automation. https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L14

Reply:
@jan-janssen I believe that an "asset" file on GH does not have the checksum change issue that we have with a just tagged releases. Also, now that grayskull can create recipes directly from GH repos I would say that, with regards to automation, it is pretty much the same.

Reply:
Ok, I updated the recipe to use the GitHub release directly.